### PR TITLE
docs: Adding a minimal tutorial on exposing a WebSocket server

### PR DESCRIPTION
**Pull request by draychev**
We saw Azure customers asking questions about exposing a WebSocket server via our k8s ingress + an App Gateway - https://github.com/Azure/application-gateway-kubernetes-ingress/issues/169.

This augments the Tutorial with the lessons learned while running a WS server behind App Gateway v2 and the current iteration of the k8s ingress.

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -176,3 +176,86 @@ spec:
           serviceName: frontend
           servicePort: 80
 ```
+
+
+## Expose a WebSocket server
```
Comment (akshaysngupta) [inline docs/tutorial.md]: Add an entry in the contents section

File: docs/tutorial.md [hunk 2]
```diff
@@ -176,3 +177,86 @@ spec:
...
+  rules:
+    - host: ws.contoso.com
+      http:
+        paths:
+          - backend:
+              serviceName: websocket-app-service
+              servicePort: 80
+```
+
+Given that all the prerequisites are fulfilled, and you have an App Gateway controlled by a K8s Ingress in your AKS, the deployment above would result in a WebSockets server exposed on port 80 of your App Gateway's public IP and the `ws.contoso.com` domain.
```
Comment (akshaysngupta) [inline docs/tutorial.md]: Should we describe that the above webserver is configured to run websockets on `/ws` while `/` or `/health` runs an https server returning `200 OK` ?

Comment (asridharan) [reply docs/tutorial.md]: 👍 . This is pretty important once we get the custom health probes documentation in.

Comment (asridharan) [reply docs/tutorial.md]: Looks like @akshaysngupta point above is covered in the section below?

**Pull request merged**